src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/VmrDependencyUpdate.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.DotNet.Darc.Models.VirtualMonoRepo;
+
+#nullable enable
+namespace Microsoft.DotNet.DarcLib.VirtualMonoRepo;
+
+public record VmrDependencyUpdate(
```

Comment:
'd be nice to have a short comment to a public record

Reply:
Done